mattermost-plugin/server/manifest.go

```diff
@@ -20,7 +20,7 @@ const manifestStr = `
   "release_notes_url": "https://github.com/mattermost/focalboard/releases",
   "icon_path": "assets/starter-template-icon.svg",
   "version": "0.9.0",
-  "min_server_version": "5.38.0",
+  "min_server_version": "6.0.0",
```

Comment:
@jespino, do we need to manually update the min_server_version in `manifest.go` as well? I just searched for "5.38" and found it here.

Reply:
It shouldn't be updated manually. But gets built when building the server. That said, the automatic build would result in the same changes you have.

Reply:
Thanks. I'll leave this in then. Maybe a question for later: Should `manifest.go` be checked in then?